src/plugins/irc-events/link.js

```diff
@@ -106,6 +108,23 @@ function parse(msg, preview, res, client) {
 
 		break;
 
+	case "audio/mpeg3":
+	case "audio/x-mpeg-3":
+	case "audio/mpeg":
+	case "audio/x-mpeg":
+	case "audio/midi":
+	case "audio/x-mid":
+	case "audio/x-midi":
+	case "audio/ogg":
+		if (res.size > (Helper.config.prefetchMaxImageSize * 4028)) {
```

Comment:
need to change this

Reply:
Why not a new `prefetchMaxAudioSize` defaulted to 4-8 MB?

Or deprecate the existing setting in favor of a `prefetchMaxMediaSize`.